internal/controllers/machine/machine_controller.go

```diff
@@ -278,6 +280,12 @@ func (r *Reconciler) reconcile(ctx context.Context, cluster *clusterv1.Cluster,
 		})
 	}
 
+	if _, isFallbackMachine := m.Labels[clusterv1.FallbackMachineLabel]; isFallbackMachine {
+		log.Info("Skipping reconciliation for fallback machine")
+		// Do not reconcile if the machine is a fallback machine.
```

Comment:
let's explain a bit more what a fallback machine is as devs reading this controller code might not have context